WalletWasabi.Fluent/ViewModels/Wallets/Send/PrivacySuggestionsFlyoutViewModel.cs

```diff
@@ -6,6 +6,7 @@
 using System.Threading.Tasks;
 using WalletWasabi.Blockchain.TransactionBuilding;
 using WalletWasabi.Fluent.Models.Transactions;
+using WalletWasabi.Logging;
```

Comment:
Leftover using from testing? There are other unused `using`s in this file. Remove them if you have time, but it's not a blocker.